test/parallel/test-dgram-unref.js

```diff
@@ -2,8 +2,18 @@
 const common = require('../common');
 const dgram = require('dgram');
 
-const s = dgram.createSocket('udp4');
-s.bind();
-s.unref();
+{
+  // Test the case of unref()'ing a socket with a handle.
+  const s = dgram.createSocket('udp4');
+  s.bind();
```

Comment:
The test will run without the `bind()`, but binding keeps the event loop open and causes the test to timeout if unref'ing doesn't actually work.